contracts/vesting/VestingWalletRecovery.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: UNLICENSED
+// See Forta Network License: https://github.com/forta-network/forta-contracts/blob/master/LICENSE.md
+
+pragma solidity ^0.8.9;
+
+import "./VestingWalletV1.sol";
+
+/**
+ * This contract is designed for recovery in case the beneficiary was lost.
+ */
+contract VestingWalletRecovery is VestingWalletV1 {
```

Comment:
There is also `VestingWalletV2`. I don't know if any of the affected wallets are using V2 but if so this recovery upgrade would remove V2 features.

This seems like another reason to use the "light" recovery mode. It keeps the wallet in the same version it's at (V1 or V2).